tests/GridFS/BucketFunctionalTest.php

```diff
@@ -860,6 +860,9 @@ public function testDanglingOpenWritableStream(): void
         $code = <<<'PHP'
             require '%s';
             require '%s';
+            // Don't report deprecations - if the issue exists this code will
+            // result in a fatal error
+            error_reporting(E_ALL & ~E_DEPRECATED & ~E_USER_DEPRECATED);
```

Comment:
Was this due to `disableMD5` being used below? Is that even relevant to what's being tested here? Why not remove it and allow deprecation notices to fail the test?

Reply:
The deprecation in question is about implicit nullable args in a PHPUnit class, which is loaded when we call `createTestClient` as it's in the inheritance chain. The issue is that checking for no output is the only way to test this behaviour. It goes back to #779 which fixed PHPLIB-345 - without the fix PHP fails with an autoloading failure as it already started unloading things when open resources (including streams with custom stream wrappers) are closed, leading to an autoload failure that doesn't find a class.